### PR TITLE
feat(route): add 杭州电子科技大学 计算机学院 通知

### PR DESCRIPTION
**Pull request by legr4ndk**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/hdu/cs
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [x] `Puppeteer`

### CODE REVIEW

**Review by legr4ndk — COMMENTED** [review]
add radar support

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please use v2 script standard https://github.com/DIYgod/RSSHub/discussions/8236

**Comment by legr4ndk** [comment]
My route  changed to V2 standard

**Comment by legr4ndk** [comment]
sry for commit spell mistake;(

**Comment by TonyRL** [comment]
Missing maintainer.js
```JavaScript
module.exports = {
     '/cs': ['legr4ndk'],
 };

```

File: lib/v2/hdu/cs/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    return (
+        list &&
+        list
+            .map((index, item) => {
+                item = $(item);
+                const dateTxt = item.find('.date').text();
+                const date = dateTxt.slice(1, dateTxt.length - 1);
+                return {
+                    title: item.find('a').text(),
+                    pubDate: new Date(date),
```
Comment (TonyRL) [inline lib/v2/hdu/cs/index.js:25]:
```suggestion
                    pubDate: parseDate(date),
```
use paseDate()
See https://github.com/DIYgod/RSSHub/issues/6795
https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei

File: lib/v2/hdu/cs/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    const items = await getSingleRecord();
+    const out = await Promise.all(
+        items.map(async (item) => {
+            const url = link + item.link;
+            const response = await got({
+                method: 'get',
+                url,
+                headers: {
+                    Referer: link,
+                    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3100.0 Safari/537.36 RSSHub',
```
Comment (TonyRL) [inline lib/v2/hdu/cs/index.js:43]: UA should be set via env
https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/config.js#L40

File: lib/v2/hdu/cs/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            });
+            const $ = cheerio.load(response.data);
+            const content = $('.wp_articlecontent').html();
+            const record = {
+                title: item.title,
+                link: url,
+                description: content,
+                pubDate: item.pubDate,
+            };
+            return Promise.resolve(record);
```
Comment (TonyRL) [inline lib/v2/hdu/cs/index.js:54]: Not necessary to create another variable. You can simply
```suggestion
            item.descriptioncontent = $('.wp_articlecontent').html();
            return item;
```

File: lib/v2/hdu/cs/index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            })
+            .get()
+    );
+};
+
+module.exports = async (ctx) => {
+    const items = await getSingleRecord();
+    const out = await Promise.all(
+        items.map(async (item) => {
+            const url = link + item.link;
```
Comment (TonyRL) [inline lib/v2/hdu/cs/index.js:37]: You can return a concatenated link
```suggestion
                    link: link + item.find('a').attr('href'),
                };
            })
            .get()
    );
};

module.exports = async (ctx) => {
    const items = await getSingleRecord();
    const out = await Promise.all(
        items.map(async (item) => {
```

Comment (TonyRL) [inline lib/v2/hdu/cs/index.js:37]: Please add cache using `ctx.cache.tryGet()`
Docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/hdu/cs/index.js (hunk 3 above)
Comment (legr4ndk) [reply lib/v2/hdu/cs/index.js:54]: can't get detailed passage when I replaced these codes

**Comment by TonyRL** [comment]
Please click the "Resolve" buttons

**Comment by legr4ndk** [comment]
> Please click the "Resolve" buttons

sry for missing that, this is my first try to contribute to a open source project ;(

File: lib/v2/hdu/cs/index.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            })
+            .get()
+    );
+};
+
+module.exports = async (ctx) => {
+    const items = await getSingleRecord();
+    const out = await Promise.all(
+        items.map(async (item) => {
+            await ctx.cache.get(item.link, async () => {
```
Comment (DIYgod) [inline lib/v2/hdu/cs/index.js:38]: The cached results are ignored

**Review by DIYgod — CHANGES_REQUESTED** [review]

**Pull request merged**